testing/unittests/jlibtests.cpp

```diff
@@ -188,6 +189,60 @@ class JlibTraceTest : public CppUnit::TestFixture
         CPPUNIT_ASSERT_EQUAL_MESSAGE("Missing resource attribute detected", true, jtraceAsTree->hasProp("resources/telemetry.sdk.name"));
     }*/
 
+    //not able to programmatically test yet, but can visually inspect trace output
+    void manualTestsDeclaredSpanStartTime()
+    {
+        Owned<IProperties> emptyMockHTTPHeaders = createProperties();
+        SpanTimeStamp declaredSpanStartTime;
+        declaredSpanStartTime.now(); // must be initialized via now(), or setMSTickTime
+        MilliSleep(125);
+
+        {
+            //duration should be at least 125 milliseconds
```

Comment:
Since I squashed the commits, I wanted to point out these new comments and varying sleep times and also actual CPPUNIT_ASSERTs on uninitialized states.